docs/discord/technical-questions.md

```diff
@@ -3224,7 +3224,7 @@ Can you do this as well?
 Well, if I edit the original file and set the ball position to `-3.22 0.05 0`, the ball stays still when I start the simulation.
 
 ##### nap 06/26/2020 06:05:24
-No.  I thought it may have something to do with how the ball was contacting the ground, so I've changed the Y axis from the normal 0.5, to bring it down.  But it still does that.
+No.  I thought it may have something to do with how the ball was contacting the ground, so I've changed the y-axis from the normal 0.5, to bring it down.  But it still does that.
```

Comment:
This file shout not be changed manually, the next synchronization will cause conflicts because of this manual change.